packages/server/src/api/controllers/row.js

```diff
@@ -101,7 +107,7 @@ exports.save = async function(ctx) {
   // make sure link rows are up to date
   row = await linkRows.updateLinks({
     instanceId,
-    eventType: linkRows.EventType.ROW_SAVE,
+    eventType: linkRows.EventType.RECORD_SAVE,
```

Comment:
This looks like it might have been a merge issue or something, `ROW_SAVE` should be what it is defined as.

Reply:
Whoops - updated